hk/ti/landing_page/v1.js

```diff
@@ -1,5 +1,5 @@
 /* CUSTOM CODE */
-(function() {
+$(document).ready(function() {
```

Comment:
I think it should be **vwo_$**

Reply:
yeah im thinking about that too :D https://vwo.com/blog/ab-testing-tricks-how-to-test-additional-new-content-in-a-split-test/